adal/src/main/java/com/microsoft/aad/adal/AuthenticationException.java

```diff
@@ -33,6 +41,12 @@
 
     private ADALError mCode;
 
+    private Map<String, String> mResponseBody;
+
+    private int mServiceStatusCode = -1;
```

Comment:
can we make all these as final, or as many as we can

Reply:
The status code cannot be final. Ref Line#93.